railties/test/application/configuration_test.rb

```diff
@@ -1559,6 +1559,17 @@ def index
       assert_equal false, ActionView::Resolver.caching?
     end
 
+    test "config.force_ssl = true and config.assume_ssl = true" do
+      add_to_config <<~RUBY
+        config.assume_ssl = true
+        config.force_ssl = true
+      RUBY
+
+      assert_raise RuntimeError do
+        app "development"
```

Comment:
good to have assertion around error message "Both config.force_ssl and config.assume_ssl cannot be enabled simultaneously. The
            config.force_ssl and config.assume_ssl are mutually exclusive, consider enabling only one of them."